--- origsrc/vte-0.48.3/bindings/vala/Makefile.am	2017-04-13 05:24:49.000000000 -0500
+++ src/vte-0.48.3/bindings/vala/Makefile.am	2017-09-05 23:11:47.520735000 -0500
@@ -33,7 +33,10 @@ CLEANFILES += $(nodist_vapi_DATA)
 vte_@VTE_API_MAJOR_VERSION@_@VTE_API_MINOR_VERSION@_SOURCES = \
 	app.vala \
 	config.vapi \
-	$(builddir)/vte-$(VTE_API_VERSION).vapi \
+	$(NULL)
+
+vte_@VTE_API_MAJOR_VERSION@_@VTE_API_MINOR_VERSION@_DEPENDENCIES = \
+	$(VAPIGEN_VAPIS) \
 	$(NULL)
 
 nodist_vte_@VTE_API_MAJOR_VERSION@_@VTE_API_MINOR_VERSION@_SOURCES = \
@@ -41,10 +44,12 @@ nodist_vte_@VTE_API_MAJOR_VERSION@_@VTE_
 	$(NULL)
 
 vte_@VTE_API_MAJOR_VERSION@_@VTE_API_MINOR_VERSION@_VALAFLAGS = \
+	--vapidir=$(abs_builddir) \
 	--target-glib=2.38 \
 	--pkg=posix \
 	--pkg=gio-2.0 \
 	--pkg=gtk+-$(GTK_API_VERSION) \
+	--pkg=vte-$(VTE_API_VERSION) \
 	--gresources app.gresource.xml \
 	$(AM_VALAFLAGS)
 
